util/envDynawo.sh

```diff
@@ -1749,7 +1764,7 @@ binary_rpath_for_darwin() {
           error_exit "Directory $omc_lib_path does not exist."
         fi
         if [ -f "$lib_path" ]; then
-          cp $lib_path $omc_lib_path || error_exit "Copy of $lib_path into $omc_lib_path failed."
```

Comment:
what about the memory is full error?

Reply:
This error would not be catch. But the way it is now exit on false issues... so I prefer not catching one error rather than not being able to launch function